content/projects/oop/animals/part2/_index.md

```diff
@@ -161,3 +161,11 @@ Test -> Does cat eat food should Fail
 This section is not compulsory. If you do this we'll think you're cool.
 
 Add some extra tests for your Home class. Make sure that it makes all the right noises. And that you can't adopt the same pet twice.
+
+## Instructions for Reviewers
+
+- There should be no global variable.
+- `self` should not be overused, if a variable is only used in one place then it is not necessary to use `self`.
```

Comment:
I don't think the issue here is the number of times the variable is used but how. If a variable is only used within a given method then constantly referring to it as `self.variable` isn't necessary.